.github/workflows/build-images-base.yaml

```diff
@@ -65,21 +65,16 @@ jobs:
     environment: ${{ inputs.environment || 'release-base-images' }}
     runs-on: ubuntu-24.04
     steps:
-      - name: Checkout default branch (trusted)
+      - name: Checkout base branch (trusted)
```

Comment:
NIT: how about change the name to: `Checkout base or default branch (trusted)`?
The same for below changes.